chromium_src/chrome/browser/ui/views/frame/browser_view.cc

```diff
@@ -4,10 +4,14 @@
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 #include "brave/browser/ui/views/tabs/brave_browser_tab_strip_controller.h"
+#include "brave/browser/ui/views/tabs/brave_tab_strip.h"
 #include "brave/browser/ui/views/toolbar/brave_toolbar_view.h"
+#include "chrome/browser/ui/views/frame/tab_strip_region_view.h"
```

Comment:
Is this needed?

Reply:
Yup, tab_strip_region_view.h references `TabStrip`.
W/o this, I got undefind linke error.
```
Undefined symbols for architecture x86_64:
  "TabStripRegionView::TabStripRegionView(std::__Cr::unique_ptr<BraveTabStrip, std::__Cr::default_delete<BraveTabStrip> >)", referenced from:
      BrowserView::BrowserView(std::__Cr::unique_ptr<Browser, std::__Cr::default_delete<Browser> >) in libui.a(browser_view.o)
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

Reply:
Ah, makes sense!